charts/dapr/crds/configuration.yaml

```diff
@@ -85,9 +85,25 @@ spec:
                 description: APISpec describes the configuration for Dapr APIs.
                 properties:
                   allowed:
+                    description: List of allowed APIs. Can be used in conjunction with denied.
                     items:
-                      description: APIAccessRule describes an access rule for allowing
-                        a Dapr API to be enabled and accessible by an app.
+                      description: APIAccessRule describes an access rule for allowing or disallowing a Dapr API.
+                      properties:
+                        name:
+                          type: string
+                        protocol:
+                          type: string
+                        version:
+                          type: string
+                      required:
+                      - name
+                      - version
+                      type: object
+                    type: array
+                  denied:
+                    description: List of disallowed APIs. Can be used in conjunction with allowed.
+                    items:
+                      description: APIAccessRule describes an access rule for allowing or disallowing a Dapr API.
```

Comment:
APIAccessRule describes an access rule to allow or deny one Dapr API.

Reply:
Done